pkg/scanners/scannerv4/scannerv4.go

```diff
@@ -31,6 +32,10 @@ var (
 
 	scanTimeout     = env.ScanTimeout.DurationSetting()
 	metadataTimeout = 1 * time.Minute
+
+	// zeroProtoTimestampFromTime represents the zero value of a proto
+	// timestamp when initialized from the zero time.
+	zeroProtoTimestampFromTime, _ = protoTypes.TimestampProto(time.Time{})
```

Comment:
I figured we could just use `protoTypes.Timestamp{}`, but that's not equal to this. TIL

Reply:
Indeed, learned that during testing myself.